docs/man_pages/lib-management/kendoui.md

```diff
@@ -0,0 +1,49 @@
+kendoui
+==========
+
+Usage | Synopsis
+------|-------
+List all versions | `$ appbuilder kendoui [--verified] [--latest]`
+List Kendo UI Core versions | `$ appbuilder kendoui --core [--verified]`
+List Kendo UI Professional versions | `$ appbuilder kendoui --professional [--verified]`
+
+Lists officially supported Kendo UI Core and Kendo UI Professional versions.<% if(isHtml || isCordova) { %> The verified tag marks stable Kendo UI Service Pack releases.<% } %> 
```

Comment:
Lists the available Kendo UI Core and Kendo UI Professional packages that you can add to your project.